it/it-tests/src/test/java/util/ItUtils.java

```diff
@@ -338,9 +340,28 @@ public static ComponentNavigation getComponentNavigation(Orchestrator orchestrat
     return ComponentNavigation.parse(content);
   }
 
-  public static File findFileInClasspath(String classpathRelativeFilename) throws URISyntaxException {
-    ResourceLocation location = FileLocation.ofClasspath("/organization/IssueAssignTest/one-issue-per-file-profile.xml");
-    return new File(ItUtils.class.getResource(location.getPath()).toURI());
+  public static void restoreProfile(Orchestrator orchestrator, URL resource) {
+    restoreProfile(orchestrator, resource, null);
+  }
+
+  public static void restoreProfile(Orchestrator orchestrator, URL resource, String organization) {
+    newAdminWsClient(orchestrator)
+      .qualityProfiles()
+      .restoreProfile(
+        RestoreWsRequest.builder()
+          .setBackup(resourceToFile(resource))
+          .setOrganization(organization)
+          .build());
+  }
+
+  private static File resourceToFile(URL resource) {
+    URI uri;
+    try {
+      uri = resource.toURI();
+    } catch (URISyntaxException e) {
+      throw new IllegalArgumentException("Cannot find quality profile xml file '" + resource + "' in classpath");
```

Comment:
this method is generic, so message should not be specific to quality profiles. I suggest `Cannot find '" + resource + "' in classpath